### ISSUE TITLE
Reduce minimum characters in project key to 2

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#681]
### TLDR ###
re [discussion on Teams](https://teams.microsoft.com/l/message/19:876c3ec2e97e4186b71cbd01f5002963@thread.tacv2/1606226828478?tenantId=dcb89c10-67b8-48cc-8819-85de37e8af91&groupId=46adb6c8-8c09-409f-a333-2d131ab6aff8&parentMessageId=1606226828478&teamName=Did&channelName=General&createdTime=1606226828478)

We should reduce number of required characters project key to 2, down from 3, without compromising matching

### Discussion ###
Having no minimum amount of characters in customer and project key ruins matching in event body.  
Having a minimum of 2 characters on both project and customer key also makes false matches on anything but category likely.  
A suggested minimum of 3 characters for `CustomerKey` and 2 characters for `ProjectKey` seems to me to be a good tradeoff.  
That way we can have projects like `PZL HR` and `PZL PP`, but still maintain the integrity of the matching engine and avoid most false matches.

### Related issues ###
Derived from #680
